Software/Python/Examples/LEGO-Motor_DPS.py

```diff
@@ -39,7 +39,7 @@
         
         BP.set_motor_dps(BP.PORT_A, target)             # set the target speed for motor A in Degrees Per Second
         
-        print("Target Degrees Per Second: %d" % target)
+        print(("Motor A Target Degrees Per Second: %d" % target), "  Motor A Status: ", BP.get_motor_status(BP.PORT_A))
```

Comment:
Did you try this line in python3?  I'm not sure it will have the proper behaviour. Maybe it will because of the from future import at the top. Just a question I have